client/src/components/common/Button/index.js

```diff
@@ -16,3 +14,8 @@ export default function Button({
     </div>
   );
 }
+
+Button.propTypes = {
+  value: PropTypes.string.isRequired,
+  className: PropTypes.string.isRequired,
```

Comment:
className shouldn't be required, pls change it and in the default props just set it to empty string :)

Reply:
Done